partiql-parser/src/scanner.rs

```diff
@@ -119,13 +137,19 @@ impl<'val> PartiQLScanner<'val> {
             .consume(start_off + text.len(), pair.end_loc());
 
         let content = match pair.as_rule() {
-            Rule::Keyword => Content::Keyword(text),
-            _ => {
-                return syntax_error(
-                    format!("Unexpected rule: {:?}", pair),
-                    pair.start_loc().into(),
-                )
+            Rule::Keyword => Content::Keyword(text.to_uppercase().into()),
+            Rule::String => Content::String(normalize_string_lit(pair.as_str())),
+            Rule::Identifier => {
+                let ident_pair = pair.into_inner().exactly_one()?;
+                match ident_pair.as_rule() {
+                    Rule::NonQuotedIdentifier => Content::Identifier(ident_pair.as_str().into()),
+                    Rule::QuotedIdentifier => {
+                        Content::Identifier(normalize_quoted_ident(ident_pair.as_str()))
+                    }
+                    _ => return ident_pair.syntax_error(),
+                }
             }
+            _ => return pair.syntax_error(),
```

Comment:
Here is where I am cleaning up the token for the `Content` value.  This will likely need to be exposed as an internal API for the full parser to leverage.